repo/el7-x86_64-eng-rhel.json

```diff
@@ -1,6 +1,6 @@
 {
         "base-url": "http://download.eng.brq.redhat.com/rel-eng/repos/eng-rhel-7/x86_64/",
         "platform-id": "el7",
-        "snapshot-id": "el7-x86_64-eng-rel",
+        "snapshot-id": "el7-x86_64-eng-rhel",
```

Comment:
![image](https://user-images.githubusercontent.com/119861/228189945-6e37d7a8-07f3-420a-9ea7-502b4ed721c6.png)

Reply:
Yeah, I was wondering if it was worth it to change this. I decided that it is, because this is a RHEL-7 repo, that won't change. But the result would be that we would need to update the repo URL when updating the snapshot date in the respective project that uses it.